metadata-etl/src/main/resources/jython/HiveTransform.py

```diff
@@ -199,12 +199,12 @@ def transform(self, input, hive_instance, hive_metadata, hive_field_metadata):
         instance_file_writer.append(dataset_instance_record)
 
         if dataset_urn not in self.dataset_dict:
-          dataset_scehma_record = DatasetSchemaRecord(table['dataset_name'], json.dumps(schema_json), json.dumps(prop_json),
-                                                    json.dumps(flds),
-                                                    dataset_urn,
-                                                    'Hive', one_db_info['type'], table['type'],
-                                                    '', (table[TableInfo.create_time] if table.has_key(
-            TableInfo.create_time) else None), (table["lastAlterTime"]) if table.has_key("lastAlterTime") else None)
+          dataset_scehma_record = DatasetSchemaRecord(table['dataset_name'], json.dumps(schema_json),
```

Comment:
Thanks for taking time for reviewing the code. Modified this part too.
For cases like the one shown below, there are about  50 other places in other files. Will clean them up along the way:
(table[TableInfo.create_time] if table.has_key( TableInfo.create_time) else None